src/main/java/com/rackspace/ceres/app/web/MetadataController.java

```diff
@@ -95,6 +100,17 @@ public MetadataController(MetadataService metadataService, Environment environme
         .getTagValues(tenantHeader, metricName, tagKey);
   }
 
+  @GetMapping("/tags")
+  public Mono<TagKeyPairResponse> getTags(
+      @RequestParam(required = false) String metricName,
+      @RequestParam(required = false) String metricGroup,
+      @RequestHeader(value = "#{appProperties.tenantHeader}") String tenantHeader) {
+    validator.validateMetricNameAndMetricGroup(metricName, metricGroup);
```

Comment:
Instead of doing it like this, can we create a validator annotation and add that to the method?
For reference:- https://github.com/Rackspace-Segment-Support/umb-ceres-ingestor/blob/9f9fb92d4f7dcb1ead6f6cd6c0be4a06fd54a74c/src/main/java/com/rackspace/monplat/ingestor/config/AppProperties.java#L40